pkgs/applications/graphics/eddy/default.nix

```diff
@@ -0,0 +1,42 @@
+{ python37Packages
+, fetchFromGitHub
+, lib
+, jre
+, qt5
+}:
+
+python37Packages.buildPythonApplication rec {
+  pname = "eddy";
+  version = "v1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "obdasystems";
+    repo = pname;
+    rev = version;
+    sha256 = "12j77bbva5py9bd57c80cmjvf8vll40h19n81h16lvv2r2r7jynh";
+  };
+
+  propagatedBuildInputs = [
+    qt5.full
+    python37Packages.setuptools
+    python37Packages.rfc3987
+    python37Packages.JPype1
+    python37Packages.pyqt5
+  ];
+
+  doCheck = false;
```

Comment:
Why are tests disabled?

Reply:
Because they don't compile. I'm getting the following error:

```
owl (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: owl (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: owl
Traceback (most recent call last):
  File "/nix/store/l7blkply3aw5cs825jzgv1pg3z8fhrzd-python3-3.7.10/lib/python3.7/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/build/source/eddy/core/owl.py", line 36, in <module>
    from PyQt5 import QtXmlPatterns
ImportError: cannot import name 'QtXmlPatterns' from 'PyQt5' (/nix/store/sxwhf70k8p1sjk3b03vbw2fms7hsppis-python3.7-PyQt5-5.15.2/lib/python3.7/site-packages/PyQt5/__init__.py)


----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (errors=1)
Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
builder for '/nix/store/ap9pgiyslm0dibija2r81rfdiq1yakc9-eddy-1.2.1.drv' failed with exit code 1
error: build of '/nix/store/ap9pgiyslm0dibija2r81rfdiq1yakc9-eddy-1.2.1.drv' failed
```

Reply:
This probably means the package is broken. The git repo linked on https://pypi.org/project/eddy/ is also 404 and the package did not receive an update since almost 4 years. It should probably not be added to nix.

Reply:
That package on pypi is not the same one as the one I want to add to nix.

Reply:
Can you please add a comment why tests are disabled?

Reply:
There's already a comment explaining why. Should I include more details ?

Reply:
My comment on why tests are disabled:
# Tests fail with: ImportError: cannot import name 'QtXmlPatterns' from 'PyQt5'

Is that enough ?